lib/bazel/query.go

```diff
@@ -40,19 +41,25 @@ var streamedBazelCommand = func(cmd *exec.Cmd) (io.Reader, chan error, error) {
 		}
 		err = cmd.Wait()
 		if err != nil {
-			errChan <- err // Don't wrap, so raw ExitError can be picked up by the caller
+			errChan <- fmt.Errorf("command failed: `%s`: %w", strings.Join(cmd.Args, " "), err)
 		}
 	}()
 
 	return pipeReader, errChan, nil
 }
 
+type QueryResult struct {
```

Comment:
So what I had mean by the log file being required: if the workspace log file is not passed in, then the workevents property is an empty list even if the targets property is not.  I think that's counter intuitive?

Maybe either document that for right now, or change the behaviour.